piton/design/chipset/rtl/packet_filter.v.pyv

```diff
@@ -325,14 +325,6 @@ for i in range(1, len(devices)):
 
 outputstr = """
                 else begin
-`ifdef PITON_FPGA_SYNTH
```

Comment:
I don't know if we should bring this in through this path. I was planning to make the change through a PR directly to OP

Reply:
up to you, I'll leave it here and we can rebase on top of that PR?